lib/screens/v2/candidate_generator/widgets/elevator_closures.ex

```diff
@@ -66,6 +66,13 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ElevatorClosures do
     end)
   end
 
+  defp routes_serving_stop(stop_id) do
+    case Route.serving_stop(stop_id) do
+      {:ok, routes} -> routes
+      :error -> []
+    end
+  end
```

Comment:
**off-topic:** maybe someday we'll have something like [`maybe` from `Data.Maybe`](https://hackage.haskell.org/package/base-4.20.0.1/docs/Data-Maybe.html#v:maybe)... _:: sigh ::_

Reply:
Can I interest you in some [Gleam](https://gleam.run/)? 😺